gqlalchemy/query_builder.py

```diff
@@ -255,13 +262,17 @@ def variable(self) -> str:
     def labels(self) -> str:
         return self._labels if self._labels is not None else ""
 
+    @property
+    def algorithm(self) -> str:
+        return self._algorithm if self._algorithm is not None else ""
```

Comment:
Can we use positive logic in ifs?

Reply:
I think it's a better practice to use something that we think would pass more often on first if, not on else. That's up for Bruno to decide, and whether people would use or not algorithms. If in doubt, use positive logic.

Reply:
I changed it to positive logic, I don't think people would use algorithms more often than not, but now it stands out from other @property getters, for variable, labels and properties which are all in negative logic, should we change them also?

Reply:
If we decide to go with the positive logic, I would change it in all other property getters too.

Reply:
Okay, you did that in the newest commit, right? This can then be resolved.